pkg/util/utils.go

```diff
@@ -76,3 +79,10 @@ func RetryAfter(attempts int, callback func() error, d time.Duration) (err error
 	}
 	return err
 }
+
+func GetLocalkubeDownloadURL(version string, filename string) string {
+	if strings.HasPrefix(version, "http://") {
+		return version
+	}
+	return fmt.Sprintf("%s%s/%s", constants.LocalkubeDownloadURLPrefix, version, filename)
```

Comment:
As discussed in https://github.com/kubernetes/minikube/pull/239/files#r71495012 can you prepend with `v` if required - can see people specifying the version without `v` prefix & this will reduce noisy issues.